gatsby-config.js

```diff
@@ -112,17 +109,24 @@ module.exports = {
         // transformations directly during node sourcing, the default implementation is to return
         // the node as is:
         // Map based on the fields settings.
-        mapNode: node => Object.keys(FILTERS).reduce((acc, cur) => ({
-          // Keep existing fields unchanged.
-          ...acc,
-
-          // Map new fields for filters based on the data settings for them.
-          // Each filter will be an array of possible values.
-          // NOTE: if the desired filter column has the same name as the
-          // original field, the original will be overwritten; this should be
-          // fine for most cases though.
-          [cur]: node[FILTERS[cur].field].split(',').map(n => n.trim()),
-        }), node),
+        mapNode: node => ({
+          // Add translated field names so we can refer to them in Gatsby
+          ...Object.keys(FIELDS).reduce((acc, cur) => ({
+            ...acc,
+            [cur]: node[FIELDS[cur].field],
```

Comment:
You can just add this line below on the existing reduce, no need for a new reduce.

Reply:
doh, fixed, thx :)

Reply:
Actually nevermind this, I hadn't noticed this was about FIELDS, not FILTERS, a reduce inside another there would just make the inner reduce run needlessly several times. Sorry about the confusion... 🤦  I undo'd your change there as it was my fault to lead you in error.